templates/ui-frameworks/svelte/field-types/u8/Slider/edit/render.hbs

```diff
@@ -0,0 +1,2 @@
+<label for="{{label}}">{{label}}</label>
+<input name=""{{label}} type="range" min="0" max="255" bind:value="{{variable_to_read}}" />
```

Comment:
```suggestion
<input name="{{label}}" type="range" min="0" max="255" bind:value={ {{variable_to_read}} } /
```

Reply:
I saw that there are more of these in other svelte slider components

Reply:
Good catch, another one that went unnoticed previously